src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx

```diff
@@ -142,6 +149,23 @@ export const SimOptionsPage = () => {
                         </button>
                     </SettingItem>
 
+                    <SettingItem name={t('Settings.SimOptions.Side')}>
+                        <SelectGroup>
+                            {defaultSideButtons.map((button) => (
+                                <SelectItem
+                                    onSelect={() => {
+                                        if (button.index !== undefined) {
+                                            setSidePlaying(button.index);
+                                        }
+                                    }}
+                                    selected={sidePlaying === button.index}
+                                >
+                                    {button.name}
+                                </SelectItem>
+                            ))}
+                        </SelectGroup>
+                    </SettingItem>
+
```

Comment:
Could you please put this before the Throttle Calibration - Calib should always be last at the buttom.
I might put it on top at one point if the page gets too long.

Reply:
Done